features/lxd.feature

```diff
@@ -152,7 +152,7 @@ Feature: LXD Pro features
   Scenario Outline: LXD guest auto-attach
     Given a `<release>` `<machine_type>` machine with ubuntu-advantage-tools installed
     # Ensure default is "off" and setup lxd
-    When I run `snap refresh lxd --channel latest/edge` with sudo
+    When I run `snap refresh lxd --channel 6/stable` with sudo
```

Comment:
In a future validation we might want to put this as `latest/stable` for better maintainability and accurate testing against what most people will have.